<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EPINBANW03CC" failures="0" tests="3" name="Default test" time="0.035" errors="0" timestamp="2023-02-12T23:38:33 IST">
  <testcase classname="extentReport.ExtentReportExample" name="@BeforeSuite LaunchBrowser" time="0.029">
    <failure type="java.lang.NoClassDefFoundError" message="io/reactivex/rxjava3/subjects/PublishSubject">
      <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:22)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:55)
at extentReport.ExtentReportExample.LaunchBrowser(ExtentReportExample.java:79)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
... 21 more
]]>
    </failure>
  </testcase> <!-- @BeforeSuite LaunchBrowser -->
  <testcase classname="extentReport.ExtentReportExample" name="OpenGoogle" time="0.0">
    <skipped/>
  </testcase> <!-- OpenGoogle -->
  <testcase classname="extentReport.ExtentReportExample" name="OpenBing" time="0.0">
    <skipped/>
  </testcase> <!-- OpenBing -->
  <testcase classname="extentReport.ExtentReportExample" name="OpenYahoo" time="0.0">
    <skipped/>
  </testcase> <!-- OpenYahoo -->
</testsuite> <!-- Default test -->
